bundle.md

```diff
@@ -14,13 +14,7 @@ This includes the following artifacts:
     This REQUIRED file MUST reside in the root of the bundle directory and MUST be named `config.json`.
     See [`config.json`](config.md) for more details.
 
-2. <a name="containerFormat02" />A directory representing the root filesystem of the container.
-    On Windows, for Windows Server containers, this directory is REQUIRED.
-    For Hyper-V containers, it MUST be omitted.
-
-    On all other platforms, this field is REQUIRED.
-
-    If set, this directory MUST be referenced by [`root`](config.md#root) within the `config.json` file.
+2. <a name="containerFormat02" />The directory referenced by [`root.path`](config.md#root), if that property is set in `config.json`.
```

Comment:
OK, How about as following:
2. <a name="containerFormat02" />container's root filesystem: the directory referenced by [`root.path`](config.md#root), if that property is set in `config.json`.

keep same format with previous entry, just a suggestion.